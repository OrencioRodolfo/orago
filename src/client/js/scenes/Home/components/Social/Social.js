import React from 'react';
import styled from 'styled-components';
import Icon from '../../../../components/Icon/Icon';

const Wrapper = styled.div`
  > i {
    margin: 0 10px;
  }
  > i:last-child {
    margin: 0;
  }
`;

const Social = () => (
  <Wrapper>
    <Icon icon="github fa-2x" />
    <Icon icon="linkedin fa-2x" />
    <Icon icon="twitter fa-2x" />
    <Icon icon="facebook fa-2x" />
  </Wrapper>
);

export default Social;
