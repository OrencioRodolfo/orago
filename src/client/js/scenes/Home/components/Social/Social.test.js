import React from 'react';
import { shallow } from 'enzyme';
import Social from './Social';

describe('Social icons component', () => {
  it('Must render properly', () => {
    const wrapper = shallow(
      <Social />,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
