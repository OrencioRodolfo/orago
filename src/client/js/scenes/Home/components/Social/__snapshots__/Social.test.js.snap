// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social icons component Must render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Social />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Link
          className=""
          url="https://github.com/OrencioRodolfo"
        >
          <Button
            icon="github fa-2x"
          />
        </Link>,
        <Link
          className=""
          url="https://www.linkedin.com/in/rodolfo-goncalves"
        >
          <Button
            icon="linkedin fa-2x"
          />
        </Link>,
        <Link
          className=""
          url="https://twitter.com/orenciorodolfo"
        >
          <Button
            icon="twitter fa-2x"
          />
        </Link>,
        <Link
          className=""
          url="https://www.facebook.com/orenciorodolfo"
        >
          <Button
            icon="facebook fa-2x"
          />
        </Link>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            icon="github fa-2x"
          />,
          "className": "",
          "url": "https://github.com/OrencioRodolfo",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "icon": "github fa-2x",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            icon="linkedin fa-2x"
          />,
          "className": "",
          "url": "https://www.linkedin.com/in/rodolfo-goncalves",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "icon": "linkedin fa-2x",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            icon="twitter fa-2x"
          />,
          "className": "",
          "url": "https://twitter.com/orenciorodolfo",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "icon": "twitter fa-2x",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            icon="facebook fa-2x"
          />,
          "className": "",
          "url": "https://www.facebook.com/orenciorodolfo",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "icon": "facebook fa-2x",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Link
            className=""
            url="https://github.com/OrencioRodolfo"
          >
            <Button
              icon="github fa-2x"
            />
          </Link>,
          <Link
            className=""
            url="https://www.linkedin.com/in/rodolfo-goncalves"
          >
            <Button
              icon="linkedin fa-2x"
            />
          </Link>,
          <Link
            className=""
            url="https://twitter.com/orenciorodolfo"
          >
            <Button
              icon="twitter fa-2x"
            />
          </Link>,
          <Link
            className=""
            url="https://www.facebook.com/orenciorodolfo"
          >
            <Button
              icon="facebook fa-2x"
            />
          </Link>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": <Button
              icon="github fa-2x"
            />,
            "className": "",
            "url": "https://github.com/OrencioRodolfo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "function",
            "props": Object {
              "icon": "github fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": <Button
              icon="linkedin fa-2x"
            />,
            "className": "",
            "url": "https://www.linkedin.com/in/rodolfo-goncalves",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "function",
            "props": Object {
              "icon": "linkedin fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": <Button
              icon="twitter fa-2x"
            />,
            "className": "",
            "url": "https://twitter.com/orenciorodolfo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "function",
            "props": Object {
              "icon": "twitter fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": <Button
              icon="facebook fa-2x"
            />,
            "className": "",
            "url": "https://www.facebook.com/orenciorodolfo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "function",
            "props": Object {
              "icon": "facebook fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
